XamarinCommunityToolkitSample.iOS/Renderers/NoLineNavigationRenderer.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Xamarin.Forms;
+using Xamarin.Forms.Platform.iOS;
+using XamarinCommunityToolkitSample.iOS.Renderers;
+using XamarinCommunityToolkitSample.Pages;
+
+[assembly: ExportRenderer(typeof(BaseNavigationPage), typeof(NoLineNavigationPageRenderer))]
+namespace XamarinCommunityToolkitSample.iOS.Renderers
+{
+    public class NoLineNavigationPageRenderer : NavigationRenderer
+    {
+        public override void ViewWillAppear(bool animated)
+        {
+            base.ViewWillAppear(animated);
+
+            if (Element is NavigationPage)
```

Comment:
I guess this check can be omitted since NavigationRenderer is designed for work with NavigationRenderer :)

Or

```
if (!(Element is NavigationPage))
    return;
```